payments-core/src/main/java/com/stripe/android/model/PaymentMethodCreateParams.kt

```diff
@@ -466,6 +478,16 @@ data class PaymentMethodCreateParams internal constructor(
         }
     }
 
+    /**
+     * Encapsulates parameters used to create [PaymentMethodCreateParams] when using Cash App Pay.
+     *
+     * **Note**: Cash App Pay is currently in beta. The API of this class is subject to change.
```

Comment:
We can also add an `@Beta` annotation for these methods, but I’m not sure if that’s overkill. Happy to hear anyone’s thoughts.

Reply:
We don't need any additional annotations or access restrictions. See more info here: https://github.com/stripe/stripe-ios/pull/2235#discussion_r1082967893